src/pr-info.ts

```diff
@@ -399,11 +398,17 @@ configSuspicious["OTHER_FILES.txt"] = contents =>
     // all path parts are not empty and not all dots
     : !contents.split(/\n/).every(line => line === "" || isRelativePath(line)) ? "bad path"
     : undefined;
-configSuspicious["tslint.json"] = contents =>
+configSuspicious["tslint.json"] = (contents, oldText) => {
     // just the required form
-    JSON.stringify(JSON.parse(contents)) !== JSON.stringify({ "extends": "dtslint/dt.json" })
-    ? "not the required form"
-    : undefined;
+    try {
+        return JSON.stringify(JSON.parse(contents)) !== JSON.stringify({ "extends": "dtslint/dt.json" }) && !jsonDiff.compare(JSON.parse(oldText || ""), JSON.parse(contents)).every(op =>
```

Comment:
I'm not too happy about this: in addition to being a super long line, it's hard to follow, and it could produce a better message.  I think that it should do this:

* If there is no previous version, barf as usual unless it's the required value.
* If there is a previous version, barf with something like "not the required form and not moving towards it" if it's not just dropping non-`extends` values.  (Or something like that.)

Reply:
Done :heavy_check_mark: